addon-test-support/load.js

```diff
@@ -19,4 +22,69 @@ export default function loadEmberExam() {
   if (window.Testem) {
     patchTestemOutput(TestLoader);
   }
+
+  testLoader = new TestLoader();
 }
+
+// loadTests() is equivalent to ember-qunit's loadTest() except this does not create a new TestLoader instance
+function loadTests() {
+  testLoader.loadModules();
+}
+
+// setupQUnitCallBacks() registers QUnit callbacks neeeded for the load-balance option.
+function setupQUnitCallbacks(qunit) {
+  if (!location.search.includes("_loadBalance")) {
+    return;
+  }
+  qunit.begin(function() {
+    return new Promise(function(resolve) {
+      Testem.once('next-module-response', function getTestModule(moduleName) {
```

Comment:
what happens if this event fails? For example, what happens if loading a test module causes an error to be thrown, how does the test runner proceed?